### PR TITLE
Choose data to display for the Tracer Log Viewer

### PR DESCRIPTION
**Pull request by richardkasongo2003**
#### PR description:
Made it possible for user to have dropdown that shows the available data files in directory

#### PR validation:
Tested Locally

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please test

File: FWCore/Services/web/transitions.js [hunk 1]
```diff
@@ -89,479 +86,545 @@ const activityToName = [  "prefetch",
...
-    const zoom_out_button = document.getElementById('zoom_out');
-    let selected_item = null;
-    selected_view.innerHTML = "Selected: [click on box in graph]";
-    let mouse_is_down = false;
-    
-    let rows = 0;
-    for( let grouping of data.transitions ) {
-        for( let slot in grouping.slots ) {
-            ++rows;
+/* const promiseOfSomeData = fetch("modules_data.json").then(r=>r.json()).then(data => {
```
Comment (Dr15Jones) [inline FWCore/Services/web/transitions.js:89]: please remove the commented out code

**Comment by Dr15Jones** [comment]
except for the left over commented out code, this looks good

**Comment by antoniovilela** [comment]
ping (to make bot change milestone)

**Comment by makortel** [comment]
@Dr15Jones Should we merge this as is and you clean up the commented out code in a subsequent PR, or do you want to do the cleanup on top of @richardkasongo2003's branch and open a separate PR?

**Comment by Dr15Jones** [comment]
Let's merge and do any additional changes in a later PR.

**Comment by makortel** [comment]
+core

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**